gateway-server/src/main/java/org/apache/knox/gateway/util/KnoxCLI.java

```diff
@@ -2570,10 +2570,16 @@ public String getUsage() {
 
   private static Properties loadBuildProperties() {
     Properties properties = new Properties();
-    try(InputStream inputStream = KnoxCLI.class.getClassLoader().getResourceAsStream( "build.properties" )) {
-      properties.load(inputStream);
+    String BUILD_PROPERTY = "build.properties";
+    PrintStream out = System.out;
```

Comment:
We already have `out` defined on line 142 so we don't have to do it here again.

Reply:
@moresandeep  Even I tried to reuse 'out' defined in line 142, but as loadBuildProperties is a static method, it is not allowing to use "public PrintStream out = System.out;" from line 142.